test/MiniBench/BenchmarkOptions.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace MiniBench
+{
+    public class BenchmarkOptions
+    {
+        public static readonly int DefaultPipelineDepth = 1;
+        public static readonly int DefaultWarmupIterations = 10;
+        public static readonly TimeSpan DefaultTestLength = TimeSpan.FromSeconds(10);
+        public static readonly int DefaultTestRuns = 1;
+
+        private CommandOption _pipelineDepth;
+        private CommandOption _testLength;
+        private CommandOption _warmupIterations;
+        private CommandOption _testRuns;
+
+        public int PipelineDepth => _pipelineDepth.HasValue() ? int.Parse(_pipelineDepth.Value()) : DefaultPipelineDepth;
```

Comment:
Looks like Pipeline depth doesn't really have an effect, at least in the CoreFX scenario, since the `WebSocket` type is single threaded so it doesn't really speed anything up to send a bunch and then wait for responses later. Still, I'll keep it here.